### PR TITLE
fix typos and remove ineffectual assignment

### PR DESCRIPTION
**Pull request by paulfantom**
SSIA

### CODE REVIEW

File: pkg/cloud/libvirt/actuators/machine/utils/domain.go [hunk 1]
```diff
@@ -497,9 +497,6 @@ func CreateDomain(name, ignKey, volumeName, hostName, networkInterfaceName, netw
 
 	glog.Infof("setDisks")
 	VolumeKey := baseVolumePath + volumeName
```
Comment (enxebre) [inline pkg/cloud/libvirt/actuators/machine/utils/domain.go]: could you also update this to use https://golang.org/pkg/path/#Join

File: pkg/cloud/libvirt/actuators/machine/utils/domain.go [hunk 2]
```diff
@@ -497,9 +497,6 @@ func CreateDomain(name, ignKey, volumeName, hostName, networkInterfaceName, netw
...
-	if volumeName == "" {
```
Comment (enxebre) [inline pkg/cloud/libvirt/actuators/machine/utils/domain.go]: we should ideally narrow down the number of parameters injected, encapsulate them into a struct, validate the inputs/error when relevant and unit test it. I understand might be out of the scope for your current work

File: pkg/cloud/libvirt/actuators/machine/utils/domain.go (hunk 1 above)
Comment (paulfantom) [reply pkg/cloud/libvirt/actuators/machine/utils/domain.go]: Done

File: pkg/cloud/libvirt/actuators/machine/utils/domain.go (hunk 2 above)
Comment (paulfantom) [reply pkg/cloud/libvirt/actuators/machine/utils/domain.go]: that would be perfect, but it is a bit out of scope

**Comment by paulfantom** [comment]
Will be merged in #92

**Pull request closed without merging**